update-help-messages.sh

```diff
@@ -0,0 +1,129 @@
+#! /bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+# This script is used to update the help messages in the docs.
+
+INCLUDE_DIR="docs/include"
+
+# Given a single command, print the help text, and wrap the output in rst style
+# code block format in a file under include directory
+function print_help_to_file() {
+
+
+    # expand all positional parameters and trim whitespace at the end
+    local cmd
+    cmd=$(echo "$*" | sed -E "s/^ +//")
+
+    # Replace spaces in the file name with dashes We print the output of
+    # `pgcopydb --help` to `pgcopydb.rst`
+    # `pgcopydb compare --help` to `compare.rst`
+    # `pgcopydb compare data --help` to `compare-data.rst`
+    local file_path
+    local help_cmd
+    if [ "${cmd}" = "" ]; then
+        help_cmd="pgcopydb --help 2>&1"
+        file_path="${INCLUDE_DIR}/pgcopydb.rst"
+    else
+        help_cmd="pgcopydb ${cmd} --help 2>&1"
+        file_path="${INCLUDE_DIR}/${cmd// /_}.rst"
+    fi
+
+    # Generate help text by running the command, removing the line with version
+    # information and adding 3 spaces at the beginning of each line
+    local help_text
+    help_text="$( eval "${help_cmd}"  |
+        sed '/.*Running pgcopydb version.*/d' |
+        sed 's/^/   /'
```

Comment:
IIRC you can use multiple sed expressions in a single sed invocation, using `sed -e 's///' -e 's///'`.